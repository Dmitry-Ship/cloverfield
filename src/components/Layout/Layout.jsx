import PropTypes from 'prop-types';
import React from 'react';
import styles from './Layout.scss';

const Layout = ({ children }) => (
  <div className={styles.layout}>
    {children}
  </div>
);

export default Layout;

Layout.propTypes = {
  children: PropTypes.any.isRequired,
};
