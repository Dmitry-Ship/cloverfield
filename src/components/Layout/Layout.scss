@value colors: "../../styles/colors.scss";
@value primary from colors;

.layout {
  background: primary;
  padding-top: 55px;
  min-height: 100vh;
}


