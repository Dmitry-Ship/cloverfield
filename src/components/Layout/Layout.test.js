import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Layout from './Layout';

describe('<Layout />', () => {
  it('should render correctly', () => {
    const tree = renderer.create(
      <Layout />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders children when passed in', () => {
    const tree = renderer.create(
      <Layout>Hello</Layout>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
