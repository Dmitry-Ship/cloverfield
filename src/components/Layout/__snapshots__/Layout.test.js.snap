// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> renders children when passed in 1`] = `
<div
  className="layout"
>
  Hello
</div>
`;

exports[`<Layout /> should render correctly 1`] = `
<div
  className="layout"
/>
`;
